test/location.coffee

```diff
@@ -607,9 +607,9 @@ test "Verify heregexes with interpolations have the right ending position", ->
   arrayEndColumn = arrayEnd[2].first_column
   for token in [comma]
     assertColumn token, arrayEndColumn - 2
-  for token in [flagsString]
-    assertColumn token, arrayEndColumn - 2, 1
-  for token in [regexCallEnd, regexEnd, fnCallEnd]
+  for token in [flagsString, regexCallEnd]
```

Comment:
By updating the `regexCallEnd` token to be marked (correctly) as `generated`, its location data now gets updated by the rewriter (along with other generated tokens)

Its location data isn't particularly significant anyway as it's not a real token